tests/components/deconz/test_climate.py

```diff
@@ -192,62 +180,41 @@ async def test_clip_climate_device(hass):
         options={deconz.gateway.CONF_ALLOW_CLIP_SENSOR: True},
         get_state_response=data,
     )
-    assert "climate.thermostat" in gateway.deconz_ids
-    assert "sensor.thermostat" not in gateway.deconz_ids
-    assert "sensor.thermostat_battery_level" in gateway.deconz_ids
-    assert "climate.presence_sensor" not in gateway.deconz_ids
-    assert "climate.clip_thermostat" in gateway.deconz_ids
-    assert len(hass.states.async_all()) == 4
-
-    thermostat = hass.states.get("climate.thermostat")
-    assert thermostat.state == "auto"
 
-    thermostat = hass.states.get("sensor.thermostat")
-    assert thermostat is None
-
-    thermostat_battery_level = hass.states.get("sensor.thermostat_battery_level")
-    assert thermostat_battery_level.state == "100"
-
-    presence_sensor = hass.states.get("climate.presence_sensor")
-    assert presence_sensor is None
+    assert len(hass.states.async_all()) == 3
+    assert hass.states.get("climate.thermostat").state == "auto"
+    assert hass.states.get("sensor.thermostat") is None
+    assert hass.states.get("sensor.thermostat_battery_level").state == "100"
+    assert hass.states.get("climate.clip_thermostat").state == "heat"
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6271139)